features/search_type.feature

```diff
@@ -0,0 +1,15 @@
+#language: fr
+
+Fonctionnalité: Chercher un type dans une glose
+
+Scénario: avec un type qui existe
+
+  Soit un document dont je suis l'auteur affiché comme glose
+  Quand je cherche le type "Ethnography/Interview"
+  Alors je peux lire uniquement "Ethnography/Interview" dans la liste des types
+
+Scénario: avec un type qui n'existe pas
+
+  Soit un document dont je suis l'auteur affiché comme glose
+  Quand je cherche le type "Existe/Pas"
```

Comment:
What about a more realistic name? e.g. "lab report"...

Reply:
The name was change to "lab report"